apps/namadillo/src/lib/query.ts

```diff
@@ -96,30 +96,32 @@ export const buildTx = async <T>(
   chain: ChainSettings,
   queryProps: T[],
   txFn: (wrapperTxProps: WrapperTxProps, props: T) => Promise<TxMsgValue>,
-  publicKeyRevealed: boolean,
-  memo?: string
+  memo?: string,
+  skipRevealPk?: boolean
 ): Promise<EncodedTxData<T>> => {
-  const { tx } = sdk;
   const wrapperTxProps = getTxProps(account, gasConfig, chain, memo);
   const txs: TxMsgValue[] = [];
   const txProps: TxProps[] = [];
 
   // Determine if RevealPK is needed:
-  if (!publicKeyRevealed) {
-    const revealPkTx = await tx.buildRevealPk(wrapperTxProps);
-    txs.push(revealPkTx);
+  if (!skipRevealPk) {
```

Comment:
just to avoid "not-not", how about renaming this to `revealPk?: boolean = true` and `if (revealPk)`?